lib/src/data/models/like.dart

```diff
@@ -1,19 +1,19 @@
 import 'package:rescado/src/data/models/animal.dart';
 
 class Like {
+  final Animal animal;
   final DateTime timestamp;
   final String? reference;
-  final Animal animal;
 
-  Like._({
+  Like({
```

Comment:
Perhaps better to stay private and make a factory constructor `fromAnimal()` that takes an `Animal` as argument (and sets `timestamp` to `DateTime.now()` in its body). This constructor can then be used to create a `Like` in `LikesController` which I see is the only place we use the default constructor.

Reply:
Did exactly that in 8d38bed7c3077fbbc8f613761114ae10f739a4c5.